setup.cfg

```diff
@@ -15,13 +15,13 @@ classifier =
     Programming Language :: Python
 [files]
 packages =
-    tools
+    oca
```

Comment:
Shouldn't it be oca-maintainer-tools to allow other future pip packages (like oca-maintainer-quality-tools)?